packages/api/tests/endpoints/providers.js

```diff
@@ -19,6 +20,7 @@ process.env.stackName = randomString();
 process.env.internal = randomString();
 let providers;
 const esIndex = randomString();
+let esClient;
 
 const testProvider = {
```

Comment:
This should be moved to a `fakeProviderFactory()` method in `packages/api/lib/testUtils.js`